### PR TITLE
User/orilevari/update dashboard readme

### PR DESCRIPTION
**Pull request by orilevari**
Updates the dashboard readme to include an image of the output debugging convolution with png output type.
Updates readme to point to newer tag (which is yet to be created since I will target the tag at this pull request merge).

### CODE REVIEW

File: Tools/WinMLDashboard/README.md [hunk 1]
```diff
@@ -75,13 +75,16 @@ To debug your model follow these steps:
...
+
+*Intermediate PNG convolutional output from an image of a [kitten](../../SharedContent/media/kitten_224.png)*
 
 
 ## Install
 
 You can install an early preview version of WinML Dashboard from here.
 
-[Download Preview Version](https://github.com/Microsoft/Windows-Machine-Learning/releases/tag/v0.5.1)
+[Download Preview Version](https://github.com/Microsoft/Windows-Machine-Learning/releases/tag/v0.6.1)
```
Comment (zhangxiang1993) [inline Tools/WinMLDashboard/README.md]: Did you talk with Kumar about the version number?

Comment (orilevari) [reply Tools/WinMLDashboard/README.md]: Yep, this is the version number we agreed on.

File: Tools/WinMLDashboard/README.md [hunk 2]
```diff
@@ -75,13 +75,16 @@ To debug your model follow these steps:
 
 <img src="./public/ConfigureDebug.png" width=800/>
 <img src="./public/DebugView.png" width=800/>
+<img src="./public/DebugPngs.png" width=800/>
```
Comment (zhangxiang1993) [inline Tools/WinMLDashboard/README.md]: Can you add like one-line description before showing the debug output?

Comment (orilevari) [reply Tools/WinMLDashboard/README.md]: There is a one line description under the image. Would you prefer that to be above the image?

Comment (zhangxiang1993) [reply Tools/WinMLDashboard/README.md]: I think so. Separate the output image and the image of how to use above with words might be a better idea.

**Pull request merged**